### PR TITLE
Add feature postal feedback

### PR DESCRIPTION
**Pull request by goeroeku**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,6 +39,10 @@
         "psalm/plugin-laravel": "^1.2"
     },
     "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/idcloudhost/laravel-mailcoach-postal-feedback"
+        },
```
Comment (riasvdv) [inline composer.json:45]: Can you publish your package on packagist so this repository isn't needed?

Comment (goeroeku) [reply composer.json:45]: ok, i will try

Comment (goeroeku) [reply composer.json]: Hi @riasvdv , I have uploaded the library to packagist as suggested. Please try to test again...

**Comment by riasvdv** [comment]
Thanks!

**Pull request merged**